docs/docs/command-line-interface.mdx

```diff
@@ -359,6 +360,34 @@ domain files:
 ```text [rasa data validate --help]
 ```
 
+## rasa data suggest nlu
+
+You can perform data augmentation by first [generating paraphrases](https://github.com/RasaHQ/paraphraser) and subsequently running the following command in order to get data suggestions that can be added to the training data:
+
+```bash
+rasa data augment nlu
+```
+
+Data augmentation is one strategy that can help produce more training data for intents that currently have little data, are frequently confused, or exhibit low performance in general. Our data augmentation strategy works by generating paraphrases for existing training examples and then evaluating whether adding a subset of these can improve intent classification performance.
```

Comment:
```suggestion
Data augmentation is one strategy that can help produce more training data for intents that currently have little data, are frequently confused, or exhibit low performance in general. Our data augmentation strategy works by feeding the generated paraphrases and then evaluating whether adding a subset of these can improve intent classification performance.
```

Reply:
Did you mean "feeding the generated paraphrases [to the classifier]" or something similar?

Reply:
I meant feeding it to the CLI tool because the tool itself doesn't generate paraphrases.